pkg/cmd/grafana-cli/commands/install_command.go

```diff
@@ -166,7 +167,7 @@ func extractFiles(body []byte, pluginName string, filePath string) error {
 		newFile := path.Join(filePath, RemoveGitBuildFromName(pluginName, zf.Name))
 
 		if zf.FileInfo().IsDir() {
-			err := os.Mkdir(newFile, 0777)
+			err := os.Mkdir(newFile, 0750)
```

Comment:
I'm not sure about this. We set very loose permissions since we dont know how people install plugins on their servers. So even if you are logged in as X when installing the plugin, the grafana user should be able to start the binary and write local logs to that folder. 

ping @xlson @daniellee

Reply:
Yeah I don't think we can currently go to `750`, but perhaps we'd be able to move to `755`. Grafana only needs to read, not write, as far as I know. @bergquist ?

Reply:
We definitely don't want to enforce for user to be a root in order to install a plugin, right? Genuinely wondering since I don't know what's the actual happy is :)

Reply:
No we don't want to enforce that. The problem is that some users install plugins with the cli when they are logged in as root (or using sudo) and by doing so the installed plugins will be owned by root. We handle this by making the files readable by all users, or currently (777), by making them modifiable by everyone.

Reply:
Gotcha!

Reply:
Switched to `755 `